tests/catch/unit/memory/hipMemset.cc

```diff
@@ -41,15 +41,15 @@ static constexpr std::initializer_list<tupletype> tableItems {
                std::make_tuple((256*1024*1024), 0xa6, 0xCAFEBABE, 0xCAFE, 0xCA)
                };
 
-enum MemsetType {
-  hipMemsetTypeDefault,
-  hipMemsetTypeD8,
-  hipMemsetTypeD16,
-  hipMemsetTypeD32
+enum memType {
```

Comment:
I think the changes in this file are possibly a search and replace mistake

Reply:
The capitalisation used here is weird. I would definitely suggest Memset over memSet.